### PR TITLE
docs(contrib) mention caveats

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Mention important contrib notes

### CODE REVIEW

File: src/content/contribute/writers-guide.md [hunk 1]
```diff
@@ -119,3 +119,19 @@ W> This is a warning.
...
+### Assumptions and simplicity
+
+Do not make assumptions when writing the documentation.
+
+```diff
+- You might already know how to optimize bundle for production...
++ As we've learned in [production guide](/guides/production/)...
+```
+
+Do not assume things are simple.
```
Comment (montogeek) [inline src/content/contribute/writers-guide.md]: I would add to this note:

> Please do not assume things are simple. Avoid words like `just`, `simply`

Comment (EugeneHlushko) [reply src/content/contribute/writers-guide.md]: Sounds good, i will add shortly

**Comment by EugeneHlushko** [comment]
Updated, please have another look @montogeek

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**